spec/models/miq_user_role_spec.rb

```diff
@@ -208,6 +209,12 @@
     end
   end
 
+  describe "#tenant_admin" do
+    it "detects tenant_admin" do
+      expect(tenant_admin_role).to be_tenant_admin_user
+    end
```

Comment:
please add test with the `super_admin_role` (should be true) and `regular_role` (should be false)